code/modules/religion/rites.dm

```diff
@@ -179,7 +176,7 @@
 		CRASH("[name]'s perform_rite had a movable atom that has somehow turned into a non-movable!")
 	var/atom/movable/movable_reltool = religious_tool
 	var/mob/living/carbon/human/rite_target
-	if(!movable_reltool?.buckled_mobs?.len)
+	if(!movable_reltool.buckled_mobs?.len)
```

Comment:
do `!length(Movable_reltool.buckled_mobs)`.
`?.len` (or `.len`) is bad practice.